src/Options.php

```diff
@@ -641,6 +641,30 @@ public function setHttpProxy(?string $httpProxy): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    /**
+     * Return the capture_silenced_errors option.
```

Comment:
This sentence should describe what the option does rather than just say what it returns. I would change it to something like `Gets whether the silenced errors should be captured or not`